framework-plugins/lisk-framework-chain-connector-plugin/src/schemas.ts

```diff
@@ -42,6 +42,28 @@ export const configSchema = {
 	},
 };
 
+export const validatorsDataSchema = {
+	$id: '/modules/bft/validatorsHashInput',
+	type: 'object',
+	required: ['activeValidators', 'certificateThreshold'],
+	properties: {
```

Comment:
`validatorsHash` is missing, we want to store where we can leave `precommitThreshold` and `prevoteThreshold` https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/engine/bft/schemas.ts#L25